cg-spec/editors_draft.html

```diff
@@ -1751,115 +1814,353 @@ <h2>Embedded Triple Expression</h2>
+          operators such as <code>=</code> and for <code>ORDER BY</code>.
+        </p>
+        <p class="box">
+          <code>xsd:boolean</code> &nbsp;
+          <code style="color:black;font-weight:bold">RDFstar-compare</code>
+          (<code style="color:black">term</code>, <code style="color:black">term</code>)
+        </p>
+        <p>The function <code>RDFstar-compare</code> returns -1,0,1, or throws an error,
+          based on the comparison of the 3 elements of an RDF-star triple term.
+        </p>
+        <div class="box">
+          <p id="defn-sparql-compare">Define:
+            <code>sparql-compare(A, B)</code> as:
+          </p>
+          <ul>
+            <li>If neither of both A and B are RDF-star triple terms,
+              compared by SPARQL 1.1 operators <code>&lt</code>, <code>=</code>, <code>&gt</code>)
+              and return that value (-1, 0, +1).<br/>
+            </li>
+            <li>If one of A and B is not an RDF-star triple terms, then error.</li>
+            <li>If <code>sparql-compare(SUBJECT(A), SUBJECT(B)) != 0</code>,
+              then return this value.</li>
+            <li>If <code>sparql-compare(PREDICATE(A), PREDICATE(B)) != 0</code>,
+              then return this value.</li>
+            <li>Return <code>sparql-compare(OBJECT(A), OBJECT(B))</code></li>
+          </ul>
+          
+          <p>
+            If both arguments are RDF-star triple terms:
+          </p>
+          <p style="margin-left:20px">
+            <code style="color:black;font-weight:bold">RDFstar-compare</code><code>(&lt;&lt;S1 P1 O1&gt;&gt;,
+              &lt;&lt;S2 P2 O2&gt;&gt;)</code>
+          </p>
+          <p>
+            is evaluated as
+          </p>
+          <p style="margin-left:20px">sparql-compare(A, B)</p>
+        </div>
+      </section>
 
-        <p>By the evaluation semantics of SPARQL-star as defined in <a href="#evaluation-semantics"></a>, the five new built-in functions defined above can be used in exactly the same manner as <a data-cite="SPARQL11-QUERY#SparqlOps">the functions defined in the SPARQL 1.1 spec</a><!--, which includes using them within GRAPH clauses that focus on querying named graphs with a queried dataset-->. For instance, the following SPARQL-star query retrieves every <a>asserted triple</a> that is both contained in the <a>default graph</a> of the queried <a>RDF-star dataset</a> and has an <a>embedded triple</a> as its <a>subject</a>.</p>
+      <section id="rdf-star-termEquals">
+        <h3>RDFterm-equal</h3>
+        <p>
+          The function <code>RDFterm-equal</code> is the default dispatch for the
+          <code>=</code> operator.
+        </p>
+        <p>
+          Extend the function defintion so that it returns <code>true</code> if term1 and term2 are both RDF-star triple
+          terms and <code>RDFterm-equal</code> is <code>true</code> for pairwise for the component RDF terms.
```

Comment:
```suggestion
          terms and <code>RDFterm-equal</code> is pairwise <code>true</code> for the component RDF terms.
```

Reply:
"for pairwise comparison of the component RDF terms."